sources/freebase.json

```diff
@@ -3,7 +3,10 @@
 	"fullname": "Freebase",
 	"name": "freebase",
 	"prefix": "fb",
-    	"key": "id",
-	"url": "https://en.wikipedia.org/wiki/Freebase",
-	"license": "http://creativecommons.org/licenses/by/2.5/"
-}
+	"key": "id",
+	"url": "https://developers.google.com/freebase/",
+	"license": "https://developers.google.com/freebase/",
+	"license_type": "CC0 / CC BY",
```

Comment:
Looks like it's CC-BY for properties, and CC0 for Wikipedia concordances. Since WOF believes concordances to always be CC0, maybe we keep the CC0 thing just in the license text, and default the license_type to CC-BY only.